_guide/tools.md

```diff
@@ -26,7 +26,7 @@ A plugin for [retext](https://github.com/retext-project/retext) that scans text
 Another plugin for [retext](https://github.com/retext-project/retext) that checks text content for possible insensitive, inconsiderate language.
 *   [Zettlr: Markdown Editor](https://www.zettlr.com/) - [Zettlr on GitHub](https://github.com/Zettlr/Zettlr) \
 Open source Markdown editor that simplifies the content creation experience by avoiding the clutter seen in many common word processing solutions.
-*   [Drupal](http://drupal.org/) is a very accessible CMS and [these modules can enhance its accessibility](https://accessibility.civicactions.com/guide/resources)
+*   [Drupal](http://drupal.org/) is a very accessible CMS and [these modules can enhance its accessibility](guide/resources#drupal-accessibility-modules)
```

Comment:
@mgifford the links need the path to be correct and also might need to link to the `.md` file as well.

Reply:
Yes. I wasn't sure if it should be just to the "guide/resources#..." or "/guide/resources#..."   
  
How would I link to the .md? Especially with a hash?

Reply:
Try by following this example from the guidebook https://github.com/CivicActions/guidebook/blob/master/common-practices-tools/security/README.md?plain=1#L21

Reply:
Ok, I think I got it..  So counter-intuitive.